pyanaconda/modules/boss/module_manager/start_modules.py

```diff
@@ -188,13 +193,13 @@ def _process_callbacks(self, module_observers):
                     continue
 
             except UnavailableModuleError:
-                # The failure of an Anaconda module is fatal.
-                if not observer.is_addon:
+                # The failure of a required module is fatal.
+                if not self._match_module(observer.service_name, self._optional):
                     raise
 
-                # The failure of an add-on module is not fatal. Remove
+                # The failure of an optional module is not fatal. Remove
                 # it from the list of available modules and continue.
-                log.warning("Skipping %s.", observer)
+                log.debug("Skip %s. The optional module has failed.", observer.service_name)
```

Comment:
Similar suggestion: "Optional module X failed to start and will not be available during the installation."

Reply:
Fixed.